RULES.md

```diff
@@ -685,6 +696,15 @@ A row in the input file has only spaces.
 ##### References:
 * [GTFS file requirements](http://gtfs.org/reference/static/#file-requirements)
 
+#### EqualShapeDistanceSameCoordinatesNotice
+
+<a name="EqualShapeDistanceSameCoordinatesNotice"/>
+
+When sorted by `shape.shape_pt_sequence`, two consecutive shape points should not have equal values for `shape_dist_traveled`.
```

Comment:
In order to give the user a bit more context, I would rephrase this to something like:
When sorted by `shape.shape_pt_sequence`, the values for `shape_dist_traveled` must increase. 
Two consecutive points with equal values for `shape_dist_traveled` and the same values for lat / lon coordinates indicate a duplicative shape point.